Document/0x05a-Platform-Overview.md

```diff
@@ -1,37 +1,36 @@
-## Android Platform Overview
+# Android Platform Overview
 
-This chapter is going to introduce Android on the architecture point of view and will provide the reader with detailed information on it's security mechanisms. Then, it will describe the structure of an Android application and will emphasize on the Inter Process Communication (IPC) mechanisms. Last, the way Android applications are published is explained to the reader.
+This chapter introduces Android from an architecture point of view. It covers four key areas:
 
-The reader is invited to have a look at the official Android developer documentation<sup>[13]</sup> for more details on the Android platform. Even if numerous examples are given in this section, in no way this guide should be considered as the only reference on the topic.
+1. Android security architecture
+2. Android application structure
+3. Inter Process Communication (IPC)
+4. Android application publishing
 
-### Android Architecture and Security Mechanisms
+The reader is invited to have a look at the official Android developer documentation[13] for more details on the Android platform. 
 
-Android is an open source platform that can be found nowadays on many devices:
+## Android Security Architecture
 
-* Mobile Phones and Tablets
-* Wearables
-* "Smart" devices in general like TVs
+Android is an open source platform found on many commonly used devices such as Mobile Phones and Tablets, Wearables, "Smart" devices in general like TVs
 
-It also offers an applicative environment that supports not only pre-installed apps on the device, but also 3rd party apps that can be downloaded from marketplaces like Google Play.
+It also offers an application environment supporting 3rd party apps that can be downloaded from marketplaces like Google Play as well as pre-installed apps on the device. 

```

Comment:
I am not sure this is an improvement. I think this sentence could use some gramatical improvement. Maybe split it in two?